### ISSUE TITLE
tenosynovitis

### ISSUE THREAD

**Issue by anastasia-coding** [obophenotype/human-phenotype-ontology#11347]
**Preferred term label:**
tenosynovitis

**Synonyms**
-----

**Definition (free text, please give PubMed ID)**
Ultrasonographic term describing tendon inflammation and characterised by hypoechoic or anechoic thickened tissue with or without fluid within the tendon sheath, which is seen in perpendicular planes and which may exhibit a Doppler signal. (PMID: 16331793)

**Parent term (use hpo.jax.org/app)**
Abnormal tendon morphology HP:0100261

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Systemic lupus erythematosus (OMIM 152700)
Systemic sclerosis (ORPHA 90291) 
Rheumatoid arthritis (OMIM 180300)

**Your nano-attribution (ORCID)**
Anastasia-Vasiliki Madenidou (https://orcid.org/0000-0002-7382-9958)

**Comment by pnrobinson** [comment]
@anastasia-coding 

Adding new term: 
- ID: HP:6001438
- label: Tenosynovitis
- parents: [Abnormal tendon morphology (HP:0100261)](https://hpo.jax.org/app/browse/term/HP:0100261)
- pmid: PMID:16331793
- def: Inflammation of the synovial sheath that surrounds a tendon.
- comment: Tenosynovitis can be ascertained as hypoechoic or anechoic thickened tissue with or without fluid within the tendon sheath, which is seen in 2 perpendicular planes and which may exhibit Doppler signal.
- orcid: orcid:0000-0002-7382-9958
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11347
- synonyms:
